src/model/element.ts

```diff
@@ -58,6 +59,14 @@ export class ScannedElement implements Resolvable {
 
 export interface Attribute extends ScannedAttribute { inheritedFrom?: string; }
 
+export class Slot {
+  name: string;
+
```

Comment:
Recommend adding a sourceRange property, that way we can do jump to definition when someone does `<div slot="foo"></div>`, as well as giving the linter more hooks to work with too.

`ParsedHtmlDocument#getSourceRangeForNode` should do all of the heavy lifting.